asar-teensy-firmware/TestScripts/Combined/MotorandProximity/MOTOR.cpp

```diff
@@ -0,0 +1,108 @@
+/* Motor.cpp
```

Comment:
Any particular reason the filename is in all caps?

Reply:
Fixed to Motor.h and Motor.cpp